app/src/main/java/org/feature/fox/coffee_counter/ui/common/MoneyAppBar.kt

```diff
@@ -10,26 +10,25 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.lifecycle.MutableLiveData
+
+class BalanceStateProvider : PreviewParameterProvider<Pair<String, Double>> {
+    override val values: Sequence<Pair<String, Double>> = sequenceOf(Pair("Beverage", 13.0))
 
-class TitleStateProvider : PreviewParameterProvider<String> {
-    override val values: Sequence<String> = sequenceOf(
-        "Beverage",
-        "History",
-    )
 }
 
 @Preview()
```

Comment:
Paranthesis should be removed.

Reply:
They can not be removed.

Reply:
Not sure, what you want to tell with this link, but they can not be removed without an error message. Did you try to checkout and remove them?

Reply:
The link shows line 35 of `BottomNavigationView.kt` where `Preview` is written without the paranthesis.
It shouldn't be possible to render the preview either way currently, because 
```kotlin
override val values: Sequence<Pair<String, Double>> = sequenceOf(Pair("Beverage", 13.0))
```
does not match the type of the `@PreviewParameter`.
Try returning a MutableLiveData<Double> to match the `@PreviewParameter`-Type.
```kotlin
class BalanceStateProvider : PreviewParameterProvider<Pair<String, MutableLiveData<Double>>> {
    override val values: Sequence<Pair<String, MutableLiveData<Double>>> =
        sequenceOf(Pair("Beverage", MutableLiveData(13.0)))
}
```

And then it should be also possible to change:
```kotlin
@Preview()
```
into:
```kotlin
@Preview
```